var/spack/repos/builtin/packages/sz/package.py

```diff
@@ -54,6 +54,8 @@ class Sz(CMakePackage):
     depends_on('hdf5', when="+hdf5")
     depends_on('netcdf-c', when="+netcdf")
 
+    patch('ctags-only-if-requested.patch', when='@2.1.8.3')
```

Comment:
@eugeneswalker thanks for the patch to the SZ spack package!

However this patch should be applied to all versions of SZ that build using cmake: i.e. when @2.1.8.1:2.1.8.3

Reply:
Will change. Thanks.